compiler/parser.go

```diff
@@ -655,23 +677,6 @@ func (l *exprListener) EnterIfExpr(ctx *gen.IfExprContext) {
 	l.expr = listener.getExpr()
 }
 
-func (l *exprListener) EnterCondExpr(ctx *gen.CondExprContext) {
```

Comment:
why did it and how's `IfExprNode` constructed now?

Reply:
I got CondIfExpr() for this block of code.  IfExprNode is resolved in EnterIfExpr()